lib/task-data/schemas/wait-sel-events.json

```diff
@@ -0,0 +1,37 @@
+{
+    "copyright": "Copyright 2017, Dell EMC, Inc.",
+    "definitions": {
+        "alertFilter": {
+            "type": "object",
+            "properties": {
+                "count": {
```

Comment:
From the unit-test, there are still other properties which are not described in schema, such as `sensorNumber`, `action`, .etc